gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -222,7 +224,6 @@ public synchronized int read(@Nonnull byte[] buf, int offset, int length) throws
             if (numRead > 0) {
               // -1 means we actually read 0 bytes, but requested at least one byte.
               totalBytesRead += numRead;
-              statistics.incrementBytesRead(numRead);
               statistics.incrementReadOps(1);
```

Comment:
will this get updated correctly?

Reply:
It is getting validated by some unit test. Added a separate integration test to verify readBytes metric.